src/win/pipe.c

```diff
@@ -577,11 +596,6 @@ int uv_pipe_bind(uv_pipe_t* handle, const char* name) {
     handle->name = NULL;
   }
 
-  if (handle->pipe.serv.accept_reqs[0].pipeHandle != INVALID_HANDLE_VALUE) {
-    CloseHandle(handle->pipe.serv.accept_reqs[0].pipeHandle);
```

Comment:
When is this handle closed now?

Reply:
There's no longer any codepaths that could reach here (actually, there wasn't before either, but that was obscured by the fact that `uv_set_pipe_handle` could return error codes from codepaths unreachable from this particular call)